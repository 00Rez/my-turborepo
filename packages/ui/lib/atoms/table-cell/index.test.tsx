import { describe, expect, it } from "vitest";
import { TableCell } from ".";
import { render } from "@testing-library/react";

describe("TableCell", () => {
  it("renders", () => {
    const { container } = render(<TableCell>Example Link</TableCell>);
    expect(container).toMatchSnapshot();
  });
});
