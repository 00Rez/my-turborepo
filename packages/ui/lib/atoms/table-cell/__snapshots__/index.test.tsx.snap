// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TableCell > renders 1`] = `
<div>
  <div
    class="table-cell "
  >
    Example Link
  </div>
</div>
`;

exports[`undefined > renders 1`] = `
<div>
  <div
    class="table-cell "
  >
    Example Link
  </div>
</div>
`;
